packages/runfiles/index.ts

```diff
@@ -1,12 +1,8 @@
-import {BAZEL_OUT_REGEX} from './paths';
-import {Runfiles} from './runfiles';
+import { Runfiles } from "./runfiles";
 
 // Re-export the `Runfiles` class. This class if the runfile helpers need to be
 // mocked for testing purposes. This is used by the linker but also publicly exposed.
-export {Runfiles};
-// Re-export a RegExp for matching `bazel-out` paths. This is used by the linker
-// but not intended for public use.
-export {BAZEL_OUT_REGEX as _BAZEL_OUT_REGEX};
```

Comment:
This is no longer used in rules_nodejs

Reply:
nit: Land in a separate PR since it is unrelated so the history of the repo mapping change is an concise as possible

Reply:
https://github.com/bazel-contrib/rules_nodejs/pull/3786